### PR TITLE
UI for multiple request issue <-> single decision issue

### PR DESCRIPTION
**Pull request by mdbenjam**
Resolves: #8175 Connects: #7965

### Description
A couple changes in this PR.

1) Copy changes
2) Adding UI to connect multiple request issues to a single decision issue.
3) Better testing for the decision issue flow

### CODE REVIEW

File: app/models/serializers/work_queue/appeal_serializer.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ class WorkQueue::AppealSerializer < ActiveModel::Serializer
   end
 
   attribute :decision_issues do
-    object.decision_issues.map do |issue|
+    object.decision_issues.uniq.map do |issue|
```
Comment (mdbenjam) [inline app/models/serializers/work_queue/appeal_serializer.rb]: There might be decision issues duplicated. This makes sure we only send each issue once.

File: client/app/queue/SelectDispositionsView.jsx [hunk 2]
```diff
@@ -27,12 +29,18 @@ import USER_ROLE_TYPES from '../../constants/USER_ROLE_TYPES.json';
...
+  marginBottom: '10px'
+});
+
 class SelectDispositionsView extends React.PureComponent {
   constructor(props) {
     super(props);
 
     this.state = {
-      openRequestIssueIds: null,
+      openRequestIssueId: null,
```
Comment (mdbenjam) [inline client/app/queue/SelectDispositionsView.jsx]: This was always an array of 1, so just made it a variable.

File: client/app/queue/SelectDispositionsView.jsx [hunk 3]
```diff
@@ -283,6 +293,49 @@ class SelectDispositionsView extends React.PureComponent {
...
+          onChange={(issue) => this.setState({
+            decisionIssue: {
+              ...decisionIssue,
+              request_issue_ids: [...decisionIssue.request_issue_ids, issue.value]
+            }
+          })}
+        />
+        {
+          connectedRequestIssues.filter((issue) => {
+            return issue.id !== openRequestIssueId;
```
Comment (mdbenjam) [inline client/app/queue/SelectDispositionsView.jsx]: Only display request issues that are not the open request issue that are able to be removed.

File: spec/feature/queue/checkout_flows_spec.rb [hunk 4]
```diff
@@ -240,10 +298,17 @@
...
+        expect(appeal.decision_issues.count).to eq(4)
         expect(appeal.decision_issues.first.description).to eq(updated_decision_issue_text)
-        expect(appeal.decision_issues.first.remand_reasons.first.code).to eq("service_treatment_records")
-        expect(appeal.decision_issues.second.remand_reasons.first.code).to eq("medical_examinations")
+
+        remand_reasons = appeal.decision_issues.where(disposition: "remanded").map do |decision|
+          decision.remand_reasons.first.code
+        end
+
+        expect(remand_reasons).to match_array(%w[service_treatment_records medical_examinations])
```
Comment (mdbenjam) [inline spec/feature/queue/checkout_flows_spec.rb]: For some reason the ordering of issues gets messed up, so we have to use some `match_array`s. But I don't think the order of decision issues matters...

File: client/COPY.json [hunk 5]
```diff
@@ -267,6 +267,8 @@
   "DECISION_ISSUE_MODAL_DESCRIPTION_EXAMPLE": "Ex: Increased rating for arthritis is granted with an evaluation of 90%",
   "DECISION_ISSUE_MODAL_DISPOSITION": "Disposition",
   "DECISION_ISSUE_MODAL_BENEFIT_TYPE": "Benefit type",
+  "DECISION_ISSUE_MODAL_CONNECTED_ISSUES_DESCRIPTION": "Do you want to add this decision to addition issues that the Veteran is contesting? This could occur when two or more contested issues receive a single decision.",
```
Comment (aroltsch) [inline client/COPY.json]: Is this text correct? I am seeing this:
image
From here: https://github.com/department-of-veterans-affairs/caseflow/issues/7966#issuecomment-448749768

Comment (mdbenjam) [reply client/COPY.json]: Good catch. Updating.

**Pull request merged**